_guides/client/basic.md

```diff
@@ -11,6 +11,16 @@ extern crate hyper;
 extern crate tokio_core;
 ```
 
+Make sure you have these dependencies listed in your ```Cargo.toml```, else you may get errors. You may need 
+to check if these are the latest versions from [Crates.io](https://crates.io)
+
+```
```

Comment:
The ` ``` ` starts a "code fence" in markdown. So:

    ```toml
    [dependencies]
    futures = "0.1"
    hyper = "0.11"
    tokio-core = "0.1"
    ```